container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend.go

```diff
@@ -389,6 +392,7 @@ func (backend *DockerKurtosisBackend) CreateLogsAggregator(
 		logsAggregatorContainer,
 		httpPortNum,
 		sinks,
+		defaultShouldTurnPersistentVolumeLogsCollectionOff,
```

Comment:
Point of confusion: CreateEngine has a new param about whether to use persistent volume (makes sense). Shouldn't `DockerKurtosisBackend.CreateLogsAggregator` have a similar one, since presumably it's being called by the Engine which has the should-use-persistent-volume value? Otherwise, why even have the parameter to `logs_aggregator_functions.CreateLogsAggregator` if it's always a static value?

Reply:
addressed in the other comment - but `CreateLogsAggregator` should be taken off KurtosisBackend interface but leaving that for another change